### PR TITLE
[stable/jenkins] Properly handle overwrite config for additional configs

### PR DESCRIPTION
**Pull request by startnow65**
#### What this PR does / why we need it:
Additional configs are only copied (overwritten) to Jenkins home when `.Values.master.overwriteConfig` is `false`. This PR updates the template to always overwrite additional configs when `.Values.master.overwriteConfig` is `true` and only copy the first time when `.Values.master.overwriteConfig` is false.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/jenkins/templates/config.yaml [hunk 1]
```diff
@@ -221,15 +221,20 @@ data:
     cp /var/jenkins_config/config.xml /var/jenkins_home;
     cp /var/jenkins_config/jenkins.CLI.xml /var/jenkins_home;
     cp /var/jenkins_config/jenkins.model.JenkinsLocationConfiguration.xml /var/jenkins_home;
-  {{- else }}
+{{- if .Values.master.additionalConfig }}
```
Comment (torstenwalter) [inline stable/jenkins/templates/config.yaml]: I suggest to use indenation as it makes it clearer where a block starts and ends.

File: stable/jenkins/templates/config.yaml [hunk 2]
```diff
@@ -221,15 +221,20 @@ data:
...
+{{- range $key, $val := .Values.master.additionalConfig }}
+    cp /var/jenkins_config/{{- $key }} /var/jenkins_home;
+{{- end }}
+{{- end }}
+{{- else }}
     yes n | cp -i /var/jenkins_config/config.xml /var/jenkins_home;
     yes n | cp -i /var/jenkins_config/jenkins.CLI.xml /var/jenkins_home;
     yes n | cp -i /var/jenkins_config/jenkins.model.JenkinsLocationConfiguration.xml /var/jenkins_home;
-  {{- if .Values.master.additionalConfig }}
```
Comment (torstenwalter) [inline stable/jenkins/templates/config.yaml]: same here I would not remove the indentation.

**Review by torstenwalter — CHANGES_REQUESTED** [review]
The content looks good.

File: stable/jenkins/templates/config.yaml (hunk 2 above)
Comment (startnow65) [reply stable/jenkins/templates/config.yaml]: Thanks @torstenwalter I've added the indentations

**Pull request merged**